setup.py

```diff
@@ -63,8 +63,11 @@ def get_requirements(file: Path) -> List[str]:
     requirements = []
     with file.open("r") as fh:
         for line in fh.readlines():
-            if line.startswith("#"):
+            if line.startswith("#") or line.strip() == "":
                 continue
+            elif line.startswith("-r"):
+                filename = line[3:].strip()
```

Comment:
If we move strip above, `line[3:]` can be written in L70 without variable `filename`.

Reply:
There is no performance improvement by removing variables.

At first you remove documentation from code and you complicate reviews, as it's unclear what information is in the substring `[3:]`. At second you looking forward for a conflict with black, which will then cripple the code.

Reply:
@umarcor Oh there are 2 more strips ...

See update version.